src/test/java/com/microsoft/sqlserver/testframework/DBTable.java

```diff
@@ -361,32 +361,66 @@ public DBTable cloneSchema() {
     String populateTableSql() {
         StringJoiner sb = new StringJoiner(SPACE_CHAR);
 
-        sb.add("INSERT");
-        sb.add("INTO");
-        sb.add(escapedTableName);
-        sb.add("VALUES");
+        // Azure DW doesn't support VALUES (), (), () ... syntax
+        boolean isSqlAzureDW = false;
+        try {
+            isSqlAzureDW = TestUtils.isSqlAzureDW(null);
+        } catch (Exception e) {
+        }
+        if (isSqlAzureDW) {
+            for (int i = 0; i < totalRows; i++) {
+                sb.add("INSERT");
+                sb.add("INTO");
+                sb.add(escapedTableName);
+                sb.add("VALUES");
 
-        for (int i = 0; i < totalRows; i++) {
-            if (i != 0) {
-                sb.add(COMMA);
-            }
-            sb.add(OPEN_BRACKET);
-            for (int colNum = 0; colNum < totalColumns; colNum++) {
+                sb.add(OPEN_BRACKET);
+                for (int colNum = 0; colNum < totalColumns; colNum++) {
+
+                    // TODO: consider how to enclose data in case of preparedStatemets
```

Comment:
is there a backlog for this?

Reply:
Please do not add TODOs, create backlog items instead.

Reply:
i didn't add this..look at the original text to the left pls (i wouldn't do this, either)